examples/mixed-spmv/doc/results.dox

```diff
@@ -0,0 +1,16 @@
+<h1>Results</h1>
+The following is the expected result (omp):
+
+@code{.cpp}
+
+High Precision time(s): 1.2272200000e-05
+High Precision result norm: 1.1581546305e+01
+Low Precision time(s): 1.2360400000e-05
+Low Precision relative error: 4.4554843634e-08
+Hp * Lp -> Hp time(s): 1.2302300000e-05
+Hp * Lp -> Hp relative error: 1.4490698720e-08
+Lp * Lp -> Hp time(s): 1.2614800000e-05
+Lp * Lp -> Hp relative error: 1.4490698720e-08
+@endcode
```

Comment:
Is there always no difference like this? How about using `CUDA`? Or a more representative matrix?

Reply:
it depends on the matrix. I am not sure whether I can find the good one.

Reply:
results indeed look strange to me...